extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -288,7 +289,12 @@ public function getArticle(){
 			if ( !empty( $relatedPages ) ) {
 				$data[ 'relatedPages' ] = $relatedPages;
 			}
-		} catch (WikiaHttpException $exception) {
+
+			if ( $title->isMainPage() ) {
```

Comment:
Should we check here if current wiki is a part of our experiment? It's not very nice to slow down all the main pages if not needed.